x-pack/plugins/cases/server/client/configure/client.ts

```diff
@@ -203,17 +203,10 @@ async function get(
   }
 }
 
-async function getConnectors({
+export async function getConnectors({
```

Comment:
I'm exporting this so I can test it directly 😬  I _could_ test via the `createConfigurationSubClient` but it's a little more difficult.